docker/bookkeeper/Dockerfile

```diff
@@ -1,37 +1,51 @@
 #
-# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-FROM apache/bookkeeper:4.7.3
 
-ARG BK_VERSION=4.7.3
+#
+# Note: This file contains source code copied from the Apache BookKeeper
```

Comment:
I assume that the `4.9` revision still applies to this PR. For consistency with the `4.8.2` version we are changing to, I wonder if it is best to refer to the `4.8` branch.

Reply:
Yes, it'd have been nice to have a branch name that represented the now modified target version `4.8.2`. But modifying the remote branch (using steps such as those described [here](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/) seems disruptive to this PR and its comments/history. What do you say?

Reply:
Upon reading your comment again, I realized that your statement was really about the `apache/bookkeeper` source code reference in `Dockerfile`. 

Yes, the `4.9` revision still applies, as we have borrowed from that version of the code over here.